src/application/components/Queries/RunInExplorerButton.tsx

```diff
@@ -54,11 +36,11 @@ export const RunInExplorerButton = ({
             embeddedExplorerIFrame,
           });
           currentScreen(Screens.Explorer);
-        }}
-      >
-        <IconRun />
-        <span>Run in Explorer</span>
-      </button>
-    )
+        }
+      }}
+    >
+      <RunIcon />
```

Comment:
Let's explicitly skip this icon for SRs (some will narrate "image" even though the SVG itself is not accessible):

```suggestion
      <RunIcon aria-hidden="true" />
```

Reply:
Done! [6b8719e](https://github.com/apollographql/apollo-client-devtools/pull/1180/commits/6b8719e3af93e674259e2ee0147f8994e27bc5bb)